### PR TITLE
Lesson1

### PR DESCRIPTION
**Pull request by JosephPobega**
here's the new pull request for lesson1.

### CODE REVIEW

**Comment by Pobega** [comment]
this is a duplicate of #7 , just noting that here

File: lesson1/helloworld.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"This is just a message tht says Hello Python World!"
```
Comment (Pobega) [inline lesson1/helloworld.py]: This does not work, you need three quotes for multiline comments. If you are just planning on doing one line, just use # like you did in lesson2

**Comment by Pobega** [comment]
@JoePobega Please add a shebang line to the top of the file so I can run it. Something like `#!/usr/bin/env python3`. Otherwise Python won't know what to do

File: lesson1/helloworld.py (hunk 1 above)
Comment (JosephPobega) [reply lesson1/helloworld.py]: ok

Comment (JosephPobega) [reply lesson1/helloworld.py]: i'll change it

**Comment by JosephPobega** [comment]
ok, i sucessfully the comment.

**Comment by JosephPobega** [comment]
ok,  I now added the shebang and also added another comment saying that it is ready for you to open.

File: lesson1/helloworld.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3
+"message tht says Hello Python World!" "i also added the shebang so you can run the hello world.py.
```
Comment (Pobega) [inline lesson1/helloworld.py]: These comments aren't right either, when I run it I get this

```
]$ python helloworld.py 
  File "helloworld.py", line 2
    "message tht says Hello Python World!" "i also added the shebang so you can run the hello world.py.
                                                                                                      ^
SyntaxError: EOL while scanning string literal
```

I think you need to put these in a # comment. Also, you don't need to talk to me through the comments, you can just text me or post a comment in the PR :)

**Review by Pobega — APPROVED** [review]
LGtM duder :+1:

**Pull request merged**